authmodules/TYPO3/privacyidea/Classes/PrivacyideaService.php

```diff
@@ -110,23 +113,30 @@ public function init() {
 	 * @return int authentication statuscode, one of 0, 100 and 200
 	 */
 	public function authUser(array $user) {
-		// 0 means authentication failure
-		$ret = 0;
-		$username = $this->login['uname'];
-		$password = $this->login['uident_text'];
-		$this->logger->info("try to authenticate user [$username]");
+		if ($this->extConf['privacyIDEABackend'] != 'adminOnly'
+			|| (bool)$user['admin']
+			|| $this->authInfo['loginType'] == 'FE'
+		) {
```

Comment:
I want to use the extension in our agency because we have a few admin users which have access to all our installations. Using a central privacyIDEA server I can easily manage these users.
If normal backend users are also checked via privacyIDEA I have to enable passthru and get an entry for every normal user who logs in.